docs/diagrams/LogicClassDiagram.puml

```diff
@@ -28,17 +29,20 @@ HiddenOutside ..> Logic
 
 LogicManager .right.|> Logic
 LogicManager -right->"1" AddressBookParser
-AddressBookParser ..> XYZCommand : creates >
+AddressBookParser ..> NormalCommand : creates >
+AddressBookParser ..> UndoableCommand : creates >
 
-XYZCommand -up-|> Command
+UndoableCommand -up-|> Command
+NormalCommand -up-|> Command
```

Comment:
This is how its intended to be done. However. when I load the `.puml` file in my IDE, the diagram has significant overlap in the arrows. Do you think you could adjust it to not overlap?